automated install/basic-install.sh

```diff
@@ -187,6 +187,37 @@ if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
     echo -e "${OVER}  ${TICK} ${COL_LIGHT_GREEN}Port 53 is not in use, proceeding with install${COL_NC}"
   fi
 }
+
+#creating silent port 53 check
+silent53check(){
+# Print info regarding probing for open port 53
+# Probe localhost via 127.0.0.1 for open port 53 and sent flags
+if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
+  # what process is using 53?
+  silentwho53="$(${SUDO} lsof -i :53 +c 0 | awk 'FNR==2{ print $1 }')"
```

Comment:
You should make this into a function so both 53 port checks can use it without duplication.